Documentation/distributions.md

```diff
@@ -7,6 +7,20 @@ The [CoreOS releases page](https://coreos.com/releases/) lists the version of rk
 
 If the version of rkt included in CoreOS is too old, it's fairly trivial to fetch the desired version [via a systemd unit](install-rkt-in-coreos.md)
 
+## Ubuntu/Debian
```

Comment:
Oh, I thought this was only about packaging, but you mention the installation script too. What about mentioning that:
- rkt is not currently packaged in Ubuntu.
- the script should work on Ubuntu as well (does it?)